### ISSUE TITLE
Is there any quick start or code example?

### ISSUE THREAD

**Issue by teadrinker2015** [arlac77/fetch-rate-limit-util#516]

**Comment by brandonsturgeon** [comment]
Agreed, a few usage examples would be nice. I'm a lazy developer looking for a quick solution, I need to see how easy this is or I'm gonna skip it 👍

**Comment by nage1234** [comment]
Even i need code examples, otherwise, i need to bang my hand on the source code and have to understand it.. no time for these.. pls provide proper documentation or example for this
